llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -682,6 +682,106 @@ class Instruction : public sandboxir::User {
   /// For isa/dyn_cast.
   static bool classof(const sandboxir::Value *From);
 
+  /// Determine whether the no signed wrap flag is set.
+  bool hasNoUnsignedWrap() const LLVM_READONLY {
```

Comment:
I wouldn't mark these with LLVM_READONLY, who knows what the implementation might do under the hood